### PR TITLE
Clean-up and add missing mail

### PR DESCRIPTION
**Pull request by AlasDiablo**

### CODE REVIEW

File: tdm-be/src/lib/codes.ts [hunk 1]
```diff
@@ -18,8 +18,14 @@ export const ERROR_MESSAGE_ENRICHMENT_UNREACHABLE_ERROR = `L'enrichissement a re
 export const ERROR_CODE_ENRICHMENT_PAYLOAD_NOT_ACCEPTED_ERROR = '54';
 export const ERROR_MESSAGE_ENRICHMENT_PAYLOAD_NOT_ACCEPTED_ERROR = `L'enrichissement a refusé le traitement. Code d'erreur ${ERROR_CODE_ENRICHMENT_PAYLOAD_NOT_ACCEPTED_ERROR}`;
 
-export const ERROR_CODE_ENRICHMENT_UNEXPECTED_ERROR = '54';
+export const ERROR_CODE_ENRICHMENT_UNEXPECTED_ERROR = '55';
 export const ERROR_MESSAGE_ENRICHMENT_UNEXPECTED_ERROR = `Le enrichissement a rencontré une erreur inattendue. Code d'erreur ${ERROR_CODE_ENRICHMENT_UNEXPECTED_ERROR}`;
```
Comment (parmentf) [inline tdm-be/src/lib/codes.ts:22]: **typo:** ‏
```suggestion
export const ERROR_MESSAGE_ENRICHMENT_UNEXPECTED_ERROR = `L'enrichissement a rencontré une erreur inattendue. Code d'erreur ${ERROR_CODE_ENRICHMENT_UNEXPECTED_ERROR}`;
```

File: tdm-be/src/templates/processing-error.njk [hunk 2]
```diff
@@ -4,6 +4,10 @@ Vous trouverez dans ce mail un rapport d'erreur du traitement {{ processingId }}
 
 {{ errorMessage }}
 
+Vous pouvais retrouvé une description de la cause possible de l'erreur sur le lien ci-dessous :
```
Comment (parmentf) [inline tdm-be/src/templates/processing-error.njk:7]: **typo:** ‏
```suggestion
Vous pouvez retrouver une description de la cause possible de l'erreur sur le lien ci-dessous :
```

File: tdm-be/src/templates/processing-error.njk [hunk 3]
```diff
@@ -4,6 +4,10 @@ Vous trouverez dans ce mail un rapport d'erreur du traitement {{ processingId }}
...
+
+https://github.com/Inist-CNRS/tdm-factory/blob/main/ERROR_CODE_FR.md
```
Comment (parmentf) [inline tdm-be/src/templates/processing-error.njk:9]: **polish:** Let's make more MarkDown interpreters work

‏
```suggestion
<https://github.com/Inist-CNRS/tdm-factory/blob/main/ERROR_CODE_FR.md>
```

**Review by parmentf — CHANGES_REQUESTED** [review]

File: tdm-be/src/templates/processing-error.njk (hunk 3 above)
Comment (AlasDiablo) [reply tdm-be/src/templates/processing-error.njk:9]: I don't thinks this work in text email :/

**Review by parmentf — APPROVED** [review]

**Pull request merged**